docs/sections/cn_workflows.md

```diff
@@ -359,6 +359,65 @@ popper run -f sample.yml -c config.yml
 
 This runs the step `one` locally in the host and step `two` through slurm on 2 nodes.
 
+### Kubernetes
+
+Popper can be run on a cluster of multiple containers using Kubernetes as the fundamental resource manager to orchestrate the execution of a step to several nodes.
+You can get started with running Popper workflows through K8s(another way to represent Kubernetes) by following the example below.
+
+Let's consider a workflow `sample.yml` like the one shown below.
+```yaml
+version: '1'
+steps:
+- id: one
+  uses: docker://alpine:3.9
+  args: echo hello-world
+
+- id: two
+  uses: popperized/bin/sh@master
+  args: ls -l
+```
+
+To run all the steps of the workflow through kubernetes,
+use the `--resource-manager` or `-r` option of the `popper run` subcommand to specify the resource manager.
+
+```bash
+popper run -f sample.yml -r kubernetes
+```
+
+When this command is run, what happens underneath is the following: `stepname kubernetes run namespace podname`
+
+Things to note:
+
+* stepname : is the step id of the workflow
+* kubernetes run: command to run kubernetes
+* namespace: Namespaces are a way to divide cluster resources between multiple users (via resource quota).
+* podname: Group of containers
```

Comment:
We can also remove this list. What we can do is give a high-level description of what happens when a workflow runs on kubernetes. Something like:

> For a workflow, we first create a volume in the namespace associated to the kubctl config. Then, for each step in the workflow, we create a pod in the namespace, run it, and destroy it. If the step is referencing a dockerfile (instead of an image), we build it locally, push it to a registry so that kubernetes can pull it from the registry when it executes the corresponding